go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/ElrondNetwork/arwen-wasm-vm v0.5.0
 	github.com/ElrondNetwork/concurrent-map v0.1.3
 	github.com/ElrondNetwork/elrond-go-logger v1.0.4
-	github.com/ElrondNetwork/elrond-vm-common v0.3.0
+	github.com/ElrondNetwork/elrond-vm-common v0.3.3
```

Comment:
there should be no reference to elrond-vm-common

Reply:
have to be change to core/vmcommon

Reply:
was removed after go mod tidy